packages/playground/src/components/execution/input-output-table.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { ScrollArea } from '@harnessio/canary'
+import { Table, TableBody, TableCell, TableHeader, TableHead, TableRow, Text } from '@harnessio/canary'
+interface InputTableProps {
+  inputData: { name: string; value: string }[]
+  className?: string
+}
+
+export const InputOutputTable: React.FC<InputTableProps> = ({ inputData, className }) => {
+  const titleKey = inputData[0].name
+  const titleValue = inputData[0].value
+
+  const dataContent = inputData.slice(1)
+  // not sure if the header needs to be fixed
+  return (
+    <ScrollArea className="overflow-x-auto pt-4">
+      <Table className={className}>
+        <TableHeader>
+          <TableRow>
+            <TableHead>
+              <Text size={4} weight="semibold" className="text-ring sm">
```

Comment:
text sizes are not matching the mocks

Reply:
Also, why do we need to specify `size={4}` and class `sm` at the same time?